web-api/disable-deploying-dynamo-stream.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+( ! command -v jq > /dev/null ) && echo "jq must be installed on your machine." && exit 1
+[ -z "${ENV}" ] && echo "You must have ENV set in your environment" && exit 1
+[ -z "${AWS_ACCOUNT_ID}" ] && echo "You must have AWS_ACCOUNT_ID set in your environment" && exit 1
+[ -z "${DEPLOYING_COLOR}" ] && echo "You must have DEPLOYING_COLOR set in your environment" && exit 1
+
+UUID=$(aws lambda list-event-source-mappings --function-name "arn:aws:lambda:us-east-1:${AWS_ACCOUNT_ID}:function:streams_${ENV}_${DEPLOYING_COLOR}" --region us-east-1 | jq -r ".EventSourceMappings[0].UUID")
+aws lambda update-event-source-mapping --uuid "${UUID}" --region us-east-1 --no-enabled
```

Comment:
After merging, I might add a refactoring item to consolidate these two scripts to something like

`toggle-deploying-dynamo-stream-trigger.sh` that take a command line argument since the only real difference is `--enabled` and `--no-enabled`

Reply:
Yeah, I started with this approach, but decided to keep it very explicit while we iron this out and then come back and refactor.